torch/distributed/rpc/api.py

```diff
@@ -196,15 +197,19 @@ def _all_gather(obj):
                 timeout=timeout
             )
             worker_name_to_response_future_dict[follower_name] = fut
+
+        failed_followers = []
         for follower_name, fut in worker_name_to_response_future_dict.items():
             try:
                 fut.wait()
             except RuntimeError as ex:
-                logger.error(
-                    "{worker_name} failed to respond to 'Shutdown Proceed.' request in {timeout}".format(
-                        worker_name=follower_name, timeout=timeout
-                    )
-                )
+                failed_followers.append(follower_name)
```

Comment:
nit, this was before this PR: the `as ex` above can be taken out since it is not used in the exception handling.